components/TipOfTheDay/index.tsx

```diff
@@ -1,11 +1,34 @@
+import { useEffect, useState } from "react"
 import styles from "./TipOfTheDay.module.css"
+import { TIPS } from "../../data/tipsOfTheDay"
 
-export const TipOfTheDay = () => {
-	return (
-		<div className={styles.card}>
-			<h2>Short tip title</h2>
-			<p>Longer explanation of tip</p>
-			<a href="#">Name of Link</a>
-		</div>
-	)
+interface ITip {
+	title: string
+	description: string
+	tipLink: string
+}
+
+export const TipOfTheDay: React.FC = () => {
```

Comment:
```suggestion
export const TipOfTheDay = () => {
```
No need to specify the type, it's inferred by typescript